diff --git a/docker-compose/envs/common-blockscout.env b/docker-compose/envs/common-blockscout.env
index 2b23e63..5673fc3 100644
--- a/docker-compose/envs/common-blockscout.env
+++ b/docker-compose/envs/common-blockscout.env
@@ -12,8 +12,8 @@ ETHEREUM_JSONRPC_TRACE_URL=http://host.docker.internal:8545/
 # ETHEREUM_JSONRPC_ETH_CALL_URL=
 # ETHEREUM_JSONRPC_HTTP_TIMEOUT=
 # CHAIN_TYPE=
-NETWORK=
-SUBNETWORK=Awesome chain
+NETWORK=Etherlink
+SUBNETWORK=Sandbox
 LOGO=/images/blockscout_logo.svg
 # ETHEREUM_JSONRPC_WS_URL=
 ETHEREUM_JSONRPC_TRANSPORT=http
@@ -313,7 +313,7 @@ EXTERNAL_APPS=[]
 # RESTRICTED_LIST_KEY=
 SHOW_MAINTENANCE_ALERT=false
 MAINTENANCE_ALERT_MESSAGE=
-CHAIN_ID=
+CHAIN_ID=123123
 MAX_SIZE_UNLESS_HIDE_ARRAY=50
 HIDE_BLOCK_MINER=false
 DISPLAY_TOKEN_ICONS=false
diff --git a/docker-compose/envs/common-frontend.env b/docker-compose/envs/common-frontend.env
index a3dcded..5148160 100644
--- a/docker-compose/envs/common-frontend.env
+++ b/docker-compose/envs/common-frontend.env
@@ -1,8 +1,8 @@
 NEXT_PUBLIC_API_HOST=localhost
 NEXT_PUBLIC_API_PROTOCOL=http
 NEXT_PUBLIC_STATS_API_HOST=http://localhost:8080
-NEXT_PUBLIC_NETWORK_NAME=Awesome chain
-NEXT_PUBLIC_NETWORK_SHORT_NAME=Awesome chain
+NEXT_PUBLIC_NETWORK_NAME=Etherlink Sandbox
+NEXT_PUBLIC_NETWORK_SHORT_NAME=sdbx
 NEXT_PUBLIC_NETWORK_ID=5
 NEXT_PUBLIC_NETWORK_CURRENCY_NAME=Ether
 NEXT_PUBLIC_NETWORK_CURRENCY_SYMBOL=ETH
